3drepo/3drepo.io
# ISSUE #3322 - Upload a revision to a particular container

This fixes #3322 

#### Description
- Changed method of calling upload revision modal to use redux actions
- The uploads modal can be opening in both the ellipses menu and the revisions description on an empty container
- A file selection window opens up first. Exiting here will cancel the process
- If a file is selected then the upload revisions modal opens with the file and desination container already selected
- You can only pick one file in the initial file window. Afterwards you can add more files and containers in the usual way

#### Test cases
- Upload a file to a specific container
- Upload files to the specific container and extra containers
- You should not be able to add a file to a processing container
- The old method of bulk uploading files should be unaffected
